Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -757,7 +758,11 @@ defineSuite([
                 expect(tileset._root.visibility(scene.frameState.cullingVolume)).not.toEqual(CullingVolume.MASK_OUTSIDE);
             });
 
+            var spyUpdate = jasmine.createSpy('listener');
```

Comment:
You can remove the `addEventListener` block above and place this test near the others
```javascript
expect(tileset._root.visibility(scene.frameState.cullingVolume)).not.toEqual(CullingVolume.MASK_OUTSIDE);
```